UpdatedImageProcessing/UpdatedImageProcessing/Classifiers/classify_shape.py

```diff
@@ -0,0 +1,6 @@
+#from classifier import Classifier
+
+class ShapeClassifier(object):
+	"""
+	Step 5: Autonomously classify shape type.
+	"""
```

Comment:
Please remove this file. We will be integrating the polar side counter as the shape classifier.

Reply:
Thank you for removing this file.